docs/unxpq-introduction.md

```diff
@@ -7,89 +7,95 @@ displayed_sidebar: '0.3'
 
 ## What is Dagger?
 
-Dagger is an engine that lets you create devops pipelines as regular code running in regular containers, rather than proprietary YAML running on a proprietary CI/CD platform.
+Dagger is a cross-platform CI/CD engine with 3 defining features:
 
-A Dagger pipeline may be used to build, test, deploy, lint, scan, sign, transform, notify... or perhaps all of the above!
+1. Portable. Run your pipelines on any Docker-compatible container runtime. In practice this means all major development and CI environments.
+2. Scriptable. Develop pipelines in Go, Typescript, Python, or even a shell script. No niche language or proprietary YAML required.
+3. Extensible. Each pipeline has an API. Pipelines can be shared and reused across projects, teams or the entire community.
 
-Popular features include:
 
-* SDKs for (almost) all major languages
-* Run pipelines from your existing tools, instead of learning yet another CLI
-* Drop to GraphQL for complete control over your pipelines
-* Faster builds with automatic caching of every step
-* Pipelines run in regular containers, making them very portable
-* The same pipelines can run in development and CI
-* Share and reuse custom pipeline steps using extensions
+```mermaid
+graph LR;
+
+script["your script"] -. Dagger API ..-> engine["Dagger Engine"]
+
+subgraph A["your build pipeline"]
+  A1[" "] -.-> A2[" "] -.-> A3[" "]
+end
+subgraph B["your deploy pipeline"]
+  B1[" "] -.-> B2[" "] -.-> B3[" "] -.-> B4[" "]
+end
+subgraph C["your test pipeline"]
+  C1[" "] -.-> C2[" "] -.-> C3[" "] -.-> C4[" "]
+end
+engine -..-> A1 & B1 & C1
+```
 
-## Supported languages
 
-* Go (alpha)
-* Typescript / Javascript (alpha)
-* Python (experimental)
-* Bash (help wanted)
-* Powershell (help wanted)
-* Ruby (help wanted)
+This can drastically improve the experience of developing and running CI/CD pipelines:
 
-Want to write devops pipelines in another language? [Tell us about it in an issue](https://github.com/dagger/dagger/issues/new)!
+
+| | Before Dagger | After Dagger |
```

Comment:
Another suggestion here

"To document and share a pipeline"

Before: Write documents that get outdated regularly
After: Leverage the GraphQL schema to generate documentation and examples.

Reply:
👍

Reply:
Done